demo/redirect-flow-example/src/App.tsx

```diff
@@ -399,8 +405,16 @@ function App() {
       ticker: "MATIC",
       tickerName: "MATIC",
     };
-    await coreKitInstance.switchChain(newChainConfig);
-    setProvider(coreKitInstance.provider);
+    if (coreKitInstance.status === COREKIT_STATUS.LOGGED_IN) {
+      let localProvider = new EthereumSigningProvider({ config: { chainConfig: newChainConfig } });
+      localProvider.setupProvider(coreKitInstance);
+      setProvider(localProvider);
+    }
+    // or
+    // provider.sendAsync({
+    //   method: "wallet_switchEthereumChain",
+    //   params: [{ chainId: newChainConfig.chainId }],
+    // })
```

Comment:
Let's try to remove all commented out code before merging. Doesn't look good in production code...